src/test/java/Homework21.java

```diff
@@ -0,0 +1,49 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+
+public class Homework21 extends BaseTest {
+    String playlistName = "Burtoiu I";
+
+    @Test
+    public void renamePlaylist() {
+        login();
+        choosePlaylist();
+        contextClickChoosePlaylist();
+        enterPlaylistname();
+        Assert.assertTrue(doesPlaylistExist());
+
+    }
+
+    public void choosePlaylist() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)"))).click();
+
+    }
+
+    public void contextClickChoosePlaylist() {
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)")));
+        WebElement playlistElement = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        actions.contextClick(playlistElement).perform();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li[data-testid='playlist-context-menu-edit']"))).click();
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/219848929-a6f53de3-b02a-4cea-a70c-0c6c20c9a5e8.png)

Hint  `wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//nav[@class='menu playlist-item-menu']//li[1]"))).click();`